packages/docusaurus-theme-classic/src/theme/BlogListPage/index.tsx

```diff
@@ -25,7 +26,10 @@ function BlogListPage(props: Props): JSX.Element {
     <Layout title={title} description={blogDescription}>
       <div className="container margin-vert--lg">
         <div className="row">
-          <main className="col col--8 col--offset-2">
+          <div className="col col--2">
```

Comment:
Not a fan of empty divs (if `blogSidebarCount=0`), let's handle this case properly (add `col--offset-2` class if there is no sidebar).

Reply:
The problem is that it requires to do this check everywhere 🤪 
I'd rather encapsulate it and keep a constant layout, makes code simpler for same results.
I don't think it's a bad practice to put an empty div for layout reasons, particularly if it has a width. It's common with flexbox to do so.

Reply:
Well, okay, although I don't really like this approach.